rslaser/pulse/pulse.py

```diff
@@ -282,10 +282,18 @@ def _wavefront(self, params, files):
                     _yStart=y_min, _yFin=y_max, _ny=ny,
                     _zStart=0., _partBeam=None)
             return
-        # calculate slice energy intensity (not energy associated with lambda)
-        sliceEnInt = params.pulseE*np.exp(-self._pulse_pos**2/(2*self.sig_s**2))
-        self.wfr = srwutil.createGsnSrcSRW(self.sigx_waist, self.sigy_waist, self.num_sig_trans, -self._pulse_pos + self.dist_waist, sliceEnInt, params.poltype, \
```

Comment:
wasn't `-self._pulse_pos + self.dist_waist` on line 287 an important part of [this issue](https://github.com/radiasoft/rslaser/commit/e9ca3ea9358dcf6dcf54a9fad29cc0d56af3a43c) might be why test is failing now

Reply:
We removed `dist_waist` from the definition of slice position, `_pulse_pos` so that the center of the pulse would be created at z=0. `dist_waist` then needed to be added to the Gaussian slice creator, `createGsnSrcSRW` to make sure that the waist occurs `dist_waist` from the center of the pulse.
This was previously a bug. Tests should be adjusted to match the updated code.